sympy/stats/crv_types.py

```diff
@@ -2449,6 +2451,93 @@ def LogLogistic(name, alpha, beta):
 
     return rv(name, LogLogisticDistribution, (alpha, beta))
 
+#-------------------------------------------------------------------------------
+#Logit-Normal distribution------------------------------------------------------
+
+class LogitNormalDistribution(SingleContinuousDistribution):
+    _argnames = ('mu', 's')
+    set = Interval(0, 1)
```

Comment:
```suggestion
    set = Interval.open(0, 1)
```
I believe `logit` isn't defined for `x = 0` and `x = 1`.